pesto-cli/pesto/ws/app.py

```diff
@@ -1,43 +1,117 @@
+    serialized = json.loads(message)
+    simplified = dict()
+    for k in mapping.keys():
+        if k != 'level' and k !='time':
+            simplified[mapping[k]] = serialized['record'][k]
+        elif k != 'time':
+            simplified[mapping[k]] = serialized['record']['level']['name']
+        else:
+            simplified[mapping[k]] = datetime.utcfromtimestamp(serialized['record']['time']['timestamp']).strftime('%Y-%m-%dT%H:%M:%S:%fZ')
 
-# API Configuration
-app.blueprint(openapi_blueprint)
-app.blueprint(swagger_blueprint)
-app.blueprint(v1)
+    # Add extra key which don't correspond to record first level of information
+    for k in serialized['record']['extra']:
+        simplified[k] = serialized['record']['extra']['k']
+    print(simplified, flush=True)
 
-app.config.API_VERSION = '1.0.0'
-app.config.API_TITLE = 'GeoProcessing SDK API'
-app.config.API_DESCRIPTION = 'GeoProcessing SDK API'
-app.config.API_TERMS_OF_SERVICE = 'Apache 2.0'
-app.config.API_PRODUCES_CONTENT_TYPES = ['application/json', 'mimetypes/jpeg', 'mimetypes/tiff']
-app.config.API_CONTACT_EMAIL = 'tbd@airbus.com'
+class InterceptHandler(logging.Handler):
+    """
+    Default handler from examples in loguru documentaion.
+    See https://loguru.readthedocs.io/en/stable/overview.html#entirely-compatible-with-standard-logging
+    """
+    def emit(self, record):
+        # Get corresponding Loguru level if it exists
+        try:
+            level = logger.level(record.levelname).name
+        except ValueError:
+            level = record.levelno
 
-app.config.KEEP_ALIVE = False
+        # Find caller from where originated the logged message
+        frame, depth = logging.currentframe(), 2
+        while frame.f_code.co_filename == logging.__file__:
+            frame = frame.f_back
+            depth += 1
+
+        logger.opt(depth=depth, exception=record.exc_info).log(
+            level, record.getMessage()
+        )
+
+def setup_logging():
+    logging.root.handlers = [InterceptHandler()]
+    logging.root.setLevel(level="DEBUG")
+    for name in logging.root.manager.loggerDict.keys():
+        logging.getLogger(name).handlers = []
+        logging.getLogger(name).propagate = True
+    if os.environ['PESTO_LOG_SERIALIZE'] == 'TRUE':
+        logger.configure(handlers=[{"sink": sink, "serialize": True}])
+    else:
+        logger.configure(handlers=[{"sink": sys.stdout}])
 
 TIMEOUT = 1_000_000
-app.config.REQUEST_TIMEOUT = TIMEOUT
-app.config.RESPONSE_TIMEOUT = TIMEOUT
-app.config.KEEP_ALIVE_TIMEOUT = TIMEOUT
 
+class Settings(BaseSettings):
+    app_name: str = "PESTO"
+    app_api_version: str = "1.0.0"
+    app_version: str = PESTO_VERSION
+    contact_mail: str = "pesto@airbus.com"
+    request_timeout: int = TIMEOUT
+    response_timeout: int = TIMEOUT
+    keep_alive_timeout: int = TIMEOUT
+    log_serialize = (os.environ['PESTO_LOG_SERIALIZE'] == 'TRUE')
+    log_format = os.environ['PESTO_LOG_FORMAT']
```

Comment:
Maybe I would expect `PESTO_LOG_FORMAT` to be optional when `PESTO_LOG_SERIALIZE != TRUE`. Or at least have a default value.

Reply:
Same as above